experimental/PropertyDDS/packages/property-changeset/src/helpers/typeidHelper.ts

```diff
@@ -180,9 +180,9 @@ export namespace TypeIdHelper {
      * @return The type of the referenced property
      */
     export function extractReferenceTargetTypeIdFromReference(in_typeid: string): string { // in_enum
-        if (in_typeid.substr(0, 10) === "Reference<") {
+        if (in_typeid.substring(0, 10) === "Reference<") {
             // Extract the type from the TypeID
-            return in_typeid.substr(10, in_typeid.length - 11);
+            return in_typeid.substring(10, in_typeid.length - 11);
```

Comment:
Right, I messed up with the find and replace, I will revert those changes and it need to be refactored carefully. 
I wanted to change few occurrences, because `substr` is now deprecated.